### DISCUSSION TITLE
How to Query

### DISCUSSION THREAD

**Discussion by jrmurdoch** [datalust/seq-tickets#1226] [Help]
I have a log that results in the following message

"0.2011 Profit: $3.22"

the message is created via:

$"{size:N4} Profit: {profit:C}"

How can I query all the log entries where 'profit' >= 5, or size > 10, for example?

I've tried profit >= 0 with no luck

**Comment by nblumhardt** [comment]
Hi! Thanks for getting in touch.

The problem here is that your message uses string interpolation (is prefixed by `$`), while to do structured logging you need to use [message templates](https://messagetemplates.org).

E.g. the log line should be:

```csharp
log.Information("{size:N4} Profit: {profit:C}", size, profit);
```

Let me know if this helps.

Best regards,
Nick

**Reply by jrmurdoch** [reply]
Yes, that works, thanks.

**Reply by nblumhardt** [reply]
Great 👍  - thanks for the follow-up.
